cobalt/browser/h5vcc_runtime/h5vcc_runtime_impl.cc

```diff
@@ -46,4 +46,15 @@ void H5vccRuntimeImpl::GetAndClearInitialDeepLink(
   std::move(callback).Run(manager->GetAndClearDeepLink());
 }
 
+void H5vccRuntimeImpl::AddListener(
+    mojo::PendingRemote<mojom::DeepLinkListener> listener) {
+  mojo::Remote<mojom::DeepLinkListener> listener_remote;
+  listener_remote.Bind(std::move(listener));
```

Comment:
Assuming you're not crashing (on debug builds), this means we're on the IO thread, because that's where the Mojo things happen. So can you add an annotation `DCHECK_CURRENTLY_ON(BrowserThread::IO);` here plz? Actually also on `GetAndClearInitialDeepLink()`.